charter.md

```diff
@@ -228,7 +228,7 @@ standardization in an appropriate venue.
 
 The Chairs of the WebExtensions Community Group are:
 
-  * Simeon Vincent, Independent (and before at Google)
+  * Simeon Vincent, Mozilla (previously Google)
```

Comment:
Should we include more specific dates in affiliation, possibly including the Independent affiliation? That would make it easier to rate the affiilation of older comments.

Even if we don't include the date, one could also bisect the meeting notes to find the date of "official" affiliation changes.

Reply:
I have never heard of that practice in other groups, have you?

Reply:
I don't have enough experience in other groups to give an authorative answer. I asked the question, because "previously" is vague and becomes increasingly less relevant in the future. I have no issues with keeping the text as is though.

I looked up a name of someone involved in standards whose affiliation changed (Opera -> Mozilla -> Apple), and in that example only the latest/current company is mentioned.

Reply:
I think adding a date would be helpful. Different browser vendors have different opinions, such as: whether to support service worker or event page and webRequest in MV3. It can be confusing if a person's perspective completely reverses due to a change in company affiliation. Adding a date can help others understand it.

Reply:
Keeping historical information for each participant inline feels like it could get unwieldy. The Privacy Interest Group has some relevant prior art; the [About section](https://www.w3.org/2019/09/privacy-ig-charter.html#about) of their charter has a table with the date of and short description of each change.  Perhaps we could do something like this?

Reply:
I've created #584 to track adding change history to the charter.